internal/controller/components/modelregistry/modelregistry_controller.go

```diff
@@ -94,10 +87,6 @@ func (s *componentHandler) NewComponentReconciler(ctx context.Context, mgr ctrl.
 		)).
 		WithAction(deployments.NewAction()).
 		WithAction(updateStatus).
-		// must be the final action
-		WithAction(gc.NewAction(
-			gc.WithUnremovables(gvk.ServiceMeshMember),
-		)).
```

Comment:
Should the `gc.NewAction` still remain here, just without the `gc.WithUnremovables`, or is that implicitly happening if it's not explicit?